doc/GLCM.ipynb

```diff
@@ -0,0 +1,414 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Gray Level Co-occurrence Matrices (GLCM)\n",
+    "\n",
+    "In this notebook, we will demonstrate how to use Gray Level Co-occurrence Matrices (GLCM), also known as _haralick features_, to perform texture analysis with PyImageJ. To learn more on GLCM and its applications, please visit [GLCM wikipedia](https://en.wikipedia.org/wiki/Co-occurrence_matrix) page. \n",
+    "\n",
+    "In this example, we will work with a 2D image (shape: `(500, 500)`) of HeLa cells infected with HIV<sub>NL4-3</sub> expressing [mVenus](https://www.fpbase.org/protein/mvenus/) tagged Gag polyprotein. The field of view contains 4 cells, 3 of which are at _below_ the cooperative threshold for Gag assembly (_i.e._ the concentration of HIV Gag is too low to multimerize into viral particles at the plasma membrane) and 1 cell at or _above_ the assembly threshold<sup>1</sup>. This workflow extracts 6 crops from the sample image, 4 crops from each of the cells and 2 crops from the background, and computes the GLCM correlation and difference variance values.\n",
+    "\n",
+    "![Cells with labels](https://media.imagej.net/pyimagej/doc/images/glcm_cell_label.png)\n",
+    "\n",
+    "## Texture analysis with GLCM\n",
+    "\n",
+    "The GLCM operation requires 3 parameters, in addition to the input image, that need to be carefully selected. These 3 paramers are _gray levels_, _distance_ and _orientation/angle_. \n",
+    "\n",
+    "### Gray Levels\n",
+    "\n",
+    "The number of discrete intensity values to bin the image pixel data into. Typically you want to select a value that is smaller than the max gray value of the image as a power of 2. For example, if your image is 8-bit then the max gray value (_i.e._ the maximum value you can store in an 8-bit address) is 255. Powers of 2 that are smaller than 255 and are good values to select for _gray levels_ for an 8-bit image are:\n",
+    "\n",
+    "| 2<sup>_n_</sup> | Value |\n",
+    "| :---:           | :---: |\n",
+    "| 2<sup>1</sup>   | 2     |\n",
+    "| 2<sup>2</sup>   | 4     |\n",
+    "| 2<sup>3</sup>   | 8     |\n",
+    "| 2<sup>4</sup>   | 16    |\n",
+    "| 2<sup>5</sup>   | 32    |\n",
+    "| 2<sup>6</sup>   | 64    |\n",
+    "| 2<sup>7</sup>   | 128   |\n",
+    "\n",
+    "### Distance\n",
+    "\n",
+    "The distance two pixels should be considered. For fine texutures select small pixel distances like 2. Conversely, for coarser details select a larger distance value \n",
+    "\n",
+    "### Orientation/Angle\n",
+    "\n",
+    "GLCM will compare patterns at given angles which can provide you information on the motion of the texture.\n",
```

Comment:
I think you can say something to the extent that:

"The angle to which we compare pixel values, using the previously specified distance, to check for co-occurrence. This can be tuned for detecting rotationally significant features, or for rotationally invariant features, co-occurrence matrices can be iteratively generated at varying angles and summed."